tests/test_metadata.py

```diff
@@ -385,7 +386,7 @@ def test_general_labels(
         # no EULA for openSUSE images
     else:
         assert (
-            labels["com.suse.lifecycle-url"].removesuffix("/")
+            re.sub(r"\/$", "", labels["com.suse.lifecycle-url"])
```

Comment:
nit: compared to the original version, `re.sub` has the potential to remove in any place, while `removesuffix` only removes if it matches the suffix.

Reply:
Nope, that's what the `$` (= end of string) is for:
```
>>> re.sub(r'\/$', '', "foo.bar.com/bar/fooo/")
'foo.bar.com/bar/fooo'
```

Reply:
can we simply list the erratic url in the tuple below? that way we don't allow a trailing slash for every of the option which might be incorrect. 

or we could simply fix the containers..

Reply:
I've fixed the containers now, so the whole shebang can be dropped.

Reply:
I missed the `$`, and I agree with @dirkmueller, better to fix the source issue.